dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/persistence/file/advanced/MoveDataMigrationOperationTest.kt

```diff
@@ -132,7 +150,7 @@ internal class MoveDataMigrationOperationTest {
     }
 
     @Test
-    fun `M retry with 500ms delay W run() {move fails once}`() {
+    fun `M not wait for real delay W run() {move fails once, time provider mocked}`() {
```

Comment:
I don't think we need to add `time provider mocked`, it is just a test setup details